ui/src/app/views/workspace/component.ts

```diff
@@ -32,7 +35,22 @@ export class WorkspaceComponent implements OnInit {
     this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
           cohortsReceived => {
             this.cohortList = cohortsReceived;
+            this.CohortEditService.list().then(
+              cohorts => {
+                for (const coho of cohorts) {
+                  this.cohortList.push(coho);
```

Comment:
Sure, let's review! Note that when you're sharing a demo link, it's helpful to use a versioned link (otherwise your demo links to a site at the default version, which gets overwritten when someone else pushes).